sbt-version-policy/src/test/scala/testpkg/VersionChangeTest.scala

```diff
@@ -0,0 +1,37 @@
+package testpkg
+
+import coursier.version.VersionCompatibility._
+import sbtversionpolicy.VersionChange
+import verify._
+
+object VersionChangeTest extends BasicTestSuite {
+  test("SemVer Spec") {
+    assert(VersionChange("0.1.0", "0.1.0", SemVerSpec) == VersionChange.Same)
+    assert(VersionChange("0.1.0", "0.1.1", SemVerSpec) == VersionChange.Prerelease)
+    assert(VersionChange("0.1.0", "0.2.0", SemVerSpec) == VersionChange.Prerelease)
+    assert(VersionChange("1.0.0", "1.0.0-M1", SemVerSpec) == VersionChange.Prerelease)
```

Comment:
```suggestion
    assert(VersionChange("1.0.0-M1", "1.0.0", SemVerSpec) == VersionChange.Prerelease)
```

(just for readability purpose: in this test you always put the “higher” version number in second)